toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
chicken and egg case! We're going to start inventing how we're going to display it based on this very file :)

Can you update the PR to this proposal and we'll revisit once we have Pontoon to play with?